src/gatling/scala/uk/gov/hmcts/reform/featuretoggle/MainSimulation.scala

```diff
@@ -5,29 +5,43 @@ import io.gatling.core.Predef._
 import io.gatling.http.Predef._
 import uk.gov.hmcts.reform.featuretoggle.actions.Create.createToggle
 import uk.gov.hmcts.reform.featuretoggle.actions.Read.readToggle
+import uk.gov.hmcts.reform.featuretoggle.actions.Delete.delete
 
 import scala.concurrent.duration._
 
 class MainSimulation extends Simulation {
 
   val config: Config = ConfigFactory.load()
 
+  private val totalToggles = config.getInt("total_toggle_count")
+  private val readDuration: FiniteDuration = 10.minutes
+  private val readDelay: FiniteDuration = 10.seconds
+  private val deleteDelay: FiniteDuration = readDelay + readDuration + 15.seconds
```

Comment:
gatling does now allow running scenarios sequentially, see need to set up the waiting period before we start deleting toggles :(